pbm/restore/logical.go

```diff
@@ -866,23 +889,19 @@ func (r *Restore) toState(ctx context.Context, status defs.Status, wait *time.Du
 	return toState(ctx, r.leadConn, status, r.name, r.nodeInfo, r.reconcileStatus, wait)
 }
 
-// dropShardedDBs drop all sharded databases present in the backup.
+// fullRestoreDBCleanup drops all databases present in the backup.
 // Backup is specified with bcp parameter.
-// For each sharded database present in the backup _shardsvrDropDatabase command
+// For each database present in the backup _shardsvrDropDatabase command
 // is used to drop the database from the config srv and all shards.
-func (r *Restore) dropShardedDBs(ctx context.Context, bcp *backup.BackupMeta) error {
+func (r *Restore) fullRestoreDBCleanup(ctx context.Context, bcp *backup.BackupMeta) error {
```

Comment:
previous name clearly communicated that this targets only Sharded environment. Maybe `fullShardedDBCleanup` (and `selShardedDBCleanup`)?